scripts/generateRules.mjs

```diff
@@ -290,27 +290,6 @@ function validateExtractedDropdowns(extractedDropdowns) {
   }
 }
 
-function validateDropdownCorrections(extractedDropdowns) {
-  // Make sure that every correction configured refers to a real value in a dropdown somewhere
-  Object.keys(dropdownCorrections).forEach(valToCorrect => {
-    for (const dropdownName in extractedDropdowns) {
-      for (const dropdownValue of extractedDropdowns[dropdownName]) {
-        if (valToCorrect == dropdownValue) {
-          return
-        }
-      }
-    }
-    // Didn't find the value anywhere? Throw an error to force us to correct it
```

Comment:
@trumanc I think this validation would prevent us from keeping corrections for uploads from old template versions, is that not correct?

I had to remove it to regenerate rules with the latest workbook, but let me know if I'm missing something.

Reply:
I wrote this in a way that will allow it to be compatible with old workbooks.
The idea is that you'll need to update the configured corrections here: https://github.com/usdigitalresponse/arpa-reporter/blob/staging/src/server/services/validation-rules.js#L24

They key of that map needs to be updated to the new correct value that is in in the workbook, and the old value needs to be added to the `allowableLegacyValues` list (creating a new list if it doesn't exist yet.

Reply:
Whoops, thanks!  Now that I read your response, I feel like you did explain it to me once already.  My bad.

I'll make another pass, updating the correction configurations.